src/docfx/lib/schema/JsonSchemaTransformer.cs

```diff
@@ -468,9 +468,9 @@ private string GetXrefHref(FilePath file, string uid, int uidCount, bool isRootL
 
                 errors.AddIfNotNull(xrefError);
 
-                if (xrefSpec != null && schema.XrefType != null && !schema.XrefType.Equals(xrefSpec.SchemaType, StringComparison.OrdinalIgnoreCase))
+                if (xrefSpec != null && schema.XrefType != null && !schema.XrefType.Contains(xrefSpec.SchemaType))
                 {
-                    errors.Add(Errors.Xref.XrefTypeInvalid(content, schema.XrefType, xrefSpec.SchemaType));
+                    errors.Add(Errors.Xref.XrefTypeInvalid(content, string.Join(",", schema.XrefType), xrefSpec.SchemaType));
```

Comment:
Pass the array and use `StringUtility.Join` in Errors.cs
```suggestion
                    errors.Add(Errors.Xref.XrefTypeInvalid(content, schema.XrefType, xrefSpec.SchemaType));
```